crates/stages/src/stages/bodies.rs

```diff
@@ -352,7 +355,7 @@ mod tests {
     }
 
     /// Same as [full_body_download] except we have made progress before
-    #[tokio::test]
+    #[test_log::test(tokio::test)]
```

Comment:
I use ` reth_tracing::init_tracing();` on tests where tracing is useful.

Reply:
sg, just wanted to test some things, forgot to remove